opencog/atoms/core/TypeUtils.h

```diff
@@ -27,13 +27,16 @@
 #include <opencog/atoms/base/Handle.h>
 #include <opencog/atoms/atom_types/types.h>
 #include <opencog/atoms/core/VariableList.h>
+#include <opencog/atoms/core/VariableSet.h>
 
 namespace opencog
 {
 /** \addtogroup grp_atomspace
  *  @{
  */
 
+// TODO: most of what is here should be moved to Variables
```

Comment:
... or not. The type utils can also be used without variables, for example to implement (finish implementing) https://wiki.opencog.org/w/TypedAtomLink